src/main/java/uk/gov/digital/ho/pttg/IpsStatisticsService.java

```diff
@@ -20,15 +20,15 @@
 
 @AllArgsConstructor
 @Slf4j
-class IpsStatisticsService {
+public class IpsStatisticsService {
 
-    static final IpsStatistics NO_STATISTICS = new IpsStatistics(LocalDate.MIN, LocalDate.MIN, Integer.MIN_VALUE,
-                                                                 Integer.MIN_VALUE, Integer.MIN_VALUE, Integer.MIN_VALUE);
+    public static final IpsStatistics NO_STATISTICS = new IpsStatistics(LocalDate.MIN, LocalDate.MIN, Integer.MIN_VALUE,
```

Comment:
Just a reminder to write some contract tests for the two endpoints once the @RestController annotation is applied.  E.g. what happens for missing / malformed parameters?